New structure (#3)
mikluke/golang-guideline: README.md

## Project mission

> We are engineers who use [go](https://golang.org) for solving technical issues every day for a long time and we have strong experience how to build performable, scalable readable, and, what's very important, simple applications with golang.

Reviewer: ```suggestion
We are engineers who use [go](https://golang.org) for solving technical issues every day for a long time and we have strong experience in building performable, scalable, readable and, what's very important, simple applications with golang.
```

> We wan't to carry our expirience on to the world to see lots of simple probjects that solve complicated tasks and were written with go-style code (not java/ruby/php/c++/etc in go).

Reviewer: ```suggestion
We want to carry our experience on to the world to see lots of simple projects that solve complicated tasks and were written with go-style code (not java/ruby/php/c++/etc in go).
```

Feel free to follow our guidelines/codestyle.

---

## How to contribute

If you want to contribute just follow next simple rules

### Follow style

We don't fix any style of texting, but your codestyle or guideline should be consistent with others.

### Use hierarchy

> Create new topics in appropriate directories. Name of directory should present a domain which covers these topics.

Reviewer: ```suggestion
Create new topics in appropriate directories. Name of the directory should present a domain which covers these topics.
```

## Project mission

We are engineers who use [go](https://golang.org) for solving technical issues every day for a long time and we have strong experience how to build performable, scalable readable, and, what's very important, simple applications with golang.

We want to carry our expirience on to the world to see lots of simple probjects that solve complicated tasks and were written with go-style code (not java/ruby/php/c++/etc in go).

Feel free to follow our guidelines/codestyle.

## Contributing

If you want to contribute just follow next simple rules.

### Follow style

We don't fix any style of texting, but your codestyle or guideline should be consistent with others.

### Use hierarchy

Create new topics in appropriate directories. Name of directory should present a domain which covers these topics.

### Ask help

If you don't know how to do just ask any maintainer for help.

---

### Contributors

> [Konstantin Shafranski](https://github.com/kostassh)

Reviewer: ```suggestion
[Kanstantsin Shafranski](https://github.com/kostassh)
```
